test/one/nio/http/VirtualHostTest.java

```diff
@@ -45,7 +48,17 @@ public static void beforeAll() throws IOException {
             server = new HttpServer(ConfigParser.parse(config, HttpServerConfig.class),
                 new Router0(), new Router1(), new Router2());
         server.start();
-        client = new HttpClient(new ConnectionString(URL));
+
+        long timeout = System.currentTimeMillis() + 1_000;
+        boolean exit = false;
+        ByteArrayOutputStream out = new ByteArrayOutputStream();
+
+        while (System.currentTimeMillis() < timeout && !exit) {
+            out.reset();
+            ThreadDumper.dump(out);
+            exit = out.toString().contains("NIO Acceptor");
```

Comment:
Even if Acceptor has started, it does not yet mean it has called `listen` on the socket.
Checking the availability with a thread dump is not a pretty idea anyway - hopefully this is not needed anymore after the latest fix.

Reply:
Totally agree, it was just a quick/ugly workaround. I would do start the same way with CDLatch. Good idea to have only sync version. Something like start(boolean sync) adds little value as the delay of sync version is going to be negligible anyway.

But I would on the other hand make start throw exception if not all acceptors can open a server socket. Maybe ssl port is already taken by a different process or something like that.

Reply:
I thought about making `start` throw `IOException`, but it's not obvious what to do when one of the acceptors fails while the other succeeds and starts serving requests.